docs/configure.rst

```diff
@@ -0,0 +1,163 @@
+=====================
+Install the Croud CLI
+=====================
+
+This page shows you how to install the `Croud CLI`_. The Croud command-line
+interface (CLI) lets you issue commands to create and manage CrateDB Cloud
+organizations, projects, and resources, and deploy CrateDB clusters.
+
+In this tutorial, the *Croud CLI tool* is reffered to as *Croud*.
+
+.. NOTE::
+
+   Croud requires `Python 3.6 or higher`_.
+
+.. SEEALSO::
+
+   For an outline of available Croud commands, take a look at the
+   `Croud CLI section`_.
+
+Install
+=======
+
+To install Croud, start your system's command-line interface and run this
+command:
+
+.. code-block:: console
+
+    sh$ python3 -m pip install --user -U croud
+
+Next, verify if the installation was successful by running:
+
+.. code-block:: console
+
+    sh$ croud -v
+
+If Croud was successfully installed, the command outputs the Croud version
+number, like so:
+
+.. code-block:: console
+
+   croud 0.16.0.
+
+Update Croud
+============
+
+We recommend you to check for regular Croud updates.
+
+You can download and install the latest version by issuing this command:
+
+.. code-block:: console
+
+    sh$ pip install -U croud
+
+Command-line options
+====================
+
+Croud supports the following command-line options:
+
++------------------------+--------------------------------------------------+
+| Argument               | Description                                      |
++------------------------+--------------------------------------------------+
+| ``-h``,                | Print the help message, then exit.               |
+| ``--help``             |                                                  |
++------------------------+--------------------------------------------------+
+| ``-v``,                | Print the program's version number, then exit.   |
+| ``--version``          |                                                  |
++------------------------+--------------------------------------------------+
+
+For instance, if you want to look up the Croud help, simply run:
+
+.. code-block:: console
+
+  sh$ croud --help
+
+.. SEEALSO::
+
+  For a full documentation of Croud commands, check out the
+  `Croud CLI section`_.
+
+.. highlights:: console
+
+========================
+Sign in to CrateDB Cloud
+========================
+
+This page shows you how to sign in to CrateDB Cloud using `Croud`_.
```

Comment:
this needs re-wording so that it fits in as a subsection of the larger "Configure" doc